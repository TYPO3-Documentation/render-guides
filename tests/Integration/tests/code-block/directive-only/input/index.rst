=================
Just a code block
=================

..  code-block::

    echo 'Hello, TYPO3';
