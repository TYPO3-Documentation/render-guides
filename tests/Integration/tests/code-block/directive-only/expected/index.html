<!-- content start -->
                <section class="section" id="just-a-code-block">
    <h1>Just a code block<a class="headerlink" href="#just-a-code-block" title="Permalink to this headline">Â¶</a></h1>

            <div translate="no">

        <pre class="code-block"><code>echo &#039;Hello, TYPO3&#039;;</code></pre>
    </div>
    </section>

        <!-- content end -->
